src/main/java/com/easyrest/pages/OwnerPanel.java

```diff
@@ -1,311 +1,41 @@
-        return new OwnerPanel(driver);
-    }
-
-    public OwnerPanel clickOnRestTagKebab() {
-
-        driver.findElement(kebabRestTag).click();
-        return new OwnerPanel(driver);
-    }
-
-    public OwnerPanel clickOnRestTagPizza() {
-
-        driver.findElement(pizzaRestTag).click();
-        return new OwnerPanel(driver);
-    }
-
-    public OwnerPanel clickOnRestTagCuisineTR() {
-
-        driver.findElement(cuisineTRRestTag).click();
-        return new OwnerPanel(driver);
-    }
-
-    public OwnerPanel clickOnRestTagVegetarian() {
-
-        driver.findElement(vegetarianRestTag).click();
-        return new OwnerPanel(driver);
-    }
-
-    public OwnerPanel clickOnRestTagSushi() {
-
-        driver.findElement(sushiRestTag).click();
-        return new OwnerPanel(driver);
-    }
-
-    public OwnerPanel clickOnRestTagFastFood() {
-
-        driver.findElement(fastFoodRestTag).click();
-        return new OwnerPanel(driver);
-    }
-
-    public OwnerPanel clickOnRestTagPub() {
-
-        driver.findElement(pubRestTag).click();
-        return new OwnerPanel(driver);
-    }
-
-    public OwnerPanel clickOnRestTagCuisineUA() {
-
-        driver.findElement(cuisineUARestTag).click();
-        return new OwnerPanel(driver);
-    }
-
-    public OwnerPanel clickOnRestTagBurgers() {
-
-        driver.findElement(burgersRestTag).click();
-        return new OwnerPanel(driver);
-    }
-
-    public OwnerPanel clickOnRestTagCuisineJP() {
-
-        driver.findElement(cuisineJPRestTag).click();
-        return new OwnerPanel(driver);
-    }
-
-    public OwnerPanel clickOnRestTagGreel() {
-
-        driver.findElement(greelRestTag).click();
-        return new OwnerPanel(driver);
-    }
-
-    public OwnerPanel clickOnRestTagCoffeTea() {
-
-        driver.findElement(coffeTeaRestTag).click();
-        return new OwnerPanel(driver);
-    }
-
-    public OwnerPanel sendToFieldRestaurantDescription(String descriptionRest) {
-        clearAndFillInput(restaurantDescriptionField, descriptionRest);
-        return new OwnerPanel(driver);
-    }
-
-    public OwnerPanel clickOnButtonAdd() {
-
-        driver.findElement(addButton).click();
-        return new OwnerPanel(driver);
-    }
-
-    public OwnerPanel clickOnButtonDeleteWaiter() {
-
-        driver.findElement(deleteWaiterButton).click();
-        return new OwnerPanel(driver);
-    }
-
-    public OwnerPanel clickOnButtonAdministratorMenu() {
-
-        driver.findElement(administratorsMenuButton).click();
-        return new OwnerPanel(driver);
-    }
-
-    public OwnerPanel clickOnButtonAddAdministrator() {
-
-        driver.findElement(addAdministratorButton).click();
-        return new OwnerPanel(driver);
-    }
-
-    public OwnerPanel clickOnButtonDeleteAdministrator() {
-
-        driver.findElement(deleteAdministratorButton).click();
-        return new OwnerPanel(driver);
-    }
-}
+        addRestaurant = new AddRestaurant(driver);
```

Comment:
do you need all of this if you have facade?

Reply:
cleaned